go/libkb/login_session.go

```diff
@@ -17,9 +20,10 @@ type LoginSession struct {
 	sessionFor      string // set by constructor
 	salt            []byte // retrieved from server, or set by WithSalt constructor
 	loginSessionB64 string
-	loginSession    []byte // decoded from above parameter
-	loaded          bool   // load state
-	cleared         bool   // clear state
+	loginSession    []byte    // decoded from above parameter
+	loaded          bool      // load state
+	cleared         bool      // clear state
+	atime           time.Time // load time
```

Comment:
`atime` is short for "access time", when it was last accessed.  I'd call this `ctime`, when it was created.